### DISCUSSION TITLE
Compatibility information about the dialog element using in modal

### DISCUSSION THREAD

**Discussion by phawitpp** [saadeghi/daisyui#2541] [Ideas / feature requests]
When I first used the modal in the dialog element way, I found that the dialog does not work on iPad. So, I found out that the dialog element cannot work on the iPadOS version below 15.4  (or it might be the same with the iOS version below 15.4, I haven't tested yet). It would be nice if you could add some information about the compatibility of the dialog element.

**Comment by saadeghi** [comment]
Browser support information added to the docs:  
https://daisyui.com/components/modal/

The checkbox method works for older browsers who don't support the `<dialog>` element:  
https://daisyui.com/components/modal/#method-2-using-a-hidden-checkbox-legacy
